docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -143,7 +152,7 @@ This property is optional; the default is ``true``.
 Protocol used to communicate with brokers.
 Valid values are: PLAINTEXT, SSL.
 
-This property is optional; default is ``PLAINTEXT``.
+This property is optional; if omitted, ``kafka.config.resources`` will be loaded.
```

Comment:
default effectively it might be still `PLAINTEXT`.

I would say that by default to the default Kafka security protocol is used (which is PLAINTEXT), but it could be overwritten with ```kafka.config.resources`` configuraiton.

Reply:
I modified it to : 

> This property is optional; default is ``PLAINTEXT``, but it can be overwritten in ``kafka.config.resources``.

Reply:
Just to be sure again, default `kafka.security.protocol` in Trino will now be null. It is implicitly set to `PLAINTEXT` if native kafka `security.protocol` is omitted.

Reply:
yes